app/test_main.py

```diff
@@ -1,3 +1,12 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_cat_years() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+    assert get_human_age(14, 14) == [0, 0]
+    assert get_human_age(15, 15) == [1, 1]
+    assert get_human_age(23, 23) == [1, 1]
+    assert get_human_age(24, 24) == [2, 2]
+    assert get_human_age(27, 27) == [2, 2]
+    assert get_human_age(28, 28) == [3, 2]
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
- Please, use `@pytest.mark.parametrize` as mentioned in the checklist
- Add test cases to check negative numbers and numbers that are different for cat age and dog age
- Test against raised error in a separate testing function in case data types different than integers are passed, e.g. strings, dicts, tuples, etc.